fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java

```diff
@@ -29,9 +35,15 @@
  */
 public class Main {
     private static final Logger logger = Logger.getLogger(Main.class.getName());
+    /**
+     * The number of nanoseconds in a second.
+     */
+    private static final double NANOS = 1000 * Constants.NANOMS;
```

Comment:
best to use TimeUnit, you get someone else to do these conversions.

Reply:
nice, I like this!

Reply:
done